templates/organization/edit.html

```diff
@@ -1,43 +1,7 @@
 {% extends "base.html" %}
 
-{% block js_media %}
-    {{ form.media.js }}
-    <script type="text/javascript">
-        window.django = {jQuery: $};
-
-        function pluralidx(count) {
-            return (count == 1) ? 0 : 1;
```

Comment:
questionable

Reply:
@quantum5 do you remember what this was for?

Reply:
Digging through the git history, it seems like I pulled in an admin widget at some point, which expected this thing. Since DMOJ was English-only back then, I shoved in this shim to make the code work.